charts/pravega/templates/tests/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ClusterRole
```

Comment:
Since these templates for test, I would prefer these be added under "pravega-operator/charts/test" instead of the current location.

Reply:
By definition, a test in a helm chart lives under the `templates/` directory and is a pod definition that specifies a container with a given command to run.
Reference : https://helm.sh/docs/developing_charts/